mmv1/products/alloydb/api.yaml

```diff
@@ -0,0 +1,413 @@
+          - !ruby/object:Api::Type::NestedObject
+            name: "quantityBasedRetention"
+            description: |
+              Quantity-based Backup retention policy to retain recent backups.
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: "count"
+                description: |
+                  The number of backups to retain.
+          - !ruby/object:Api::Type::Boolean
+            name: "enabled"
+            description: |
+              Whether automated automated backups are enabled.
+      - !ruby/object:Api::Type::NestedObject
+        name: "backupSource"
+        output: true
+        description: "Cluster created from backup."
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "backupName"
+            description: "The name of the backup resource."
+      - !ruby/object:Api::Type::NestedObject
+        name: "migrationSource"
+        output: true
+        description: "Cluster created via DMS migration."
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "hostPort"
+            description: "The host and port of the on-premises instance in host:port format"
+          - !ruby/object:Api::Type::String
+            name: "referenceId"
+            description: "Place holder for the external source identifier(e.g DMS job name) that created the cluster."
+          - !ruby/object:Api::Type::String
+            name: "sourceType"
+            description: "Type of migration source."
+  - !ruby/object:Api::Resource
+    name: "Instance"
+    self_link: "projects/{{project}}/locations/{{location}}/clusters/{{cluster_id}}/instances/{{instance_id}}"
+    base_url: "projects/{{project}}/locations/{{location}}/clusters/{{cluster_id}}/instances"
+    create_url: "projects/{{project}}/locations/{{location}}/clusters/{{cluster_id}}/instances?instanceId={{instance_id}}"
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: "A managed alloydb cluster instance."
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        "AlloyDB": "https://cloud.google.com/alloydb/docs/"
+      api: "https://cloud.google.com/alloydb/docs/reference/rest/v1beta/projects.locations.clusters.instances/create"
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: "name"
+        base_url: "{{op_id}}"
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
+          insert_minutes: 10
+          update_minutes: 10
+          delete_minutes: 10
+      result: !ruby/object:Api::OpAsync::Result
+        path: "response"
+      status: !ruby/object:Api::OpAsync::Status
+        path: "done"
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: "error"
+        message: "message"
+    parameters:
```

Comment:
It looks like this is missing the `machineConfig` field - is that intentional?

Reply:
missed it

Reply:
done